mono/metadata/Makefile.am

```diff
@@ -255,51 +247,10 @@ sgen_sources = \
 	sgen-tarjan-bridge.c		\
 	sgen-toggleref.c		\
 	sgen-toggleref.h		\
-	sgen-gc.h		\
-	sgen-conf.h		\
-	sgen-archdep.h		\
-	sgen-cardtable.c	\
-	sgen-cardtable.h	\
-	sgen-pointer-queue.c	\
-	sgen-pointer-queue.h	\
-	sgen-pinning.c	\
-	sgen-pinning.h	\
-	sgen-pinning-stats.c	\
-	sgen-workers.c	\
-	sgen-workers.h	\
-	sgen-gray.c	\
-	sgen-gray.h	\
-	sgen-major-copy-object.h \
-	sgen-minor-copy-object.h \
-	sgen-copy-object.h \
-	sgen-marksweep-scan-object-concurrent.h \
-	sgen-minor-scan-object.h \
-	sgen-marksweep-drain-gray-stack.h	\
-	sgen-protocol.h		\
-	sgen-protocol-def.h		\
-	sgen-scan-object.h	\
-	sgen-nursery-allocator.c	\
-	sgen-hash-table.c	\
-	sgen-hash-table.h	\
-	sgen-descriptor.c		\
-	sgen-descriptor.h		\
-	sgen-alloc.c		\
-	sgen-debug.c		\
-	sgen-simple-nursery.c	\
-	sgen-split-nursery.c	\
-	sgen-memory-governor.c	\
-	sgen-memory-governor.h	\
 	sgen-stw.c				\
-	sgen-fin-weak-hash.c	\
 	sgen-layout-stats.c	\
 	sgen-layout-stats.h	\
```

Comment:
These two are part of SGen, too.  It's just an oversight that they're not included in the simple client example.  Layout statistics is a compile-time option, so as long as it wasn't enabled, it would link without it.